### DISCUSSION TITLE
Display the selected date range label in the active filter indicators

### DISCUSSION THREAD

**Discussion by malle-pietje** [malzariey/filament-daterangepicker-filter#124] [Ideas]
Thanks a lot for this useful plugin! I've worked with the jQuery library at lot in non-Filament projects, very happy we can now leverage this plugin in our Filament projects 👍 

Trying to now customise the filter indicator using the `indicateUsing()` method but I can't find a way to get the current selected date range label. Is that possible or not?

**Comment by malzariey** [comment]
Check the current withIndicator code

https://github.com/malzariey/filament-daterangepicker-filter/blob/e5af6623a907f9dcba4c64f0257126afaffdac8a/src/Filters/DateRangeFilter.php#L127-L144

**Comment by malle-pietje** [comment]
Thanks. I had already looked at that and I would like to replace `$datesString` with the label of the selected date range, e.g “Today”. How do I achieve that?

**Reply by malzariey** [reply]
You would need to compare the string date with all ranges to replace the string

**Reply by malle-pietje** [reply]
Ok, understood. I was hoping that would be available at that point.
